qemu/tests/cfg/balloon_check.cfg

```diff
@@ -54,6 +54,16 @@
             check_image = yes
         - balloon-stop_continue:
             sub_test_after_balloon = "stop_continue"
+        - iommu_enabled:
+            only q35
+            only balloon_evict_and_enlarge
+            no WinXP WinVista Win7 Win8 Win8.1 Win2000 Win2003
+            no Win2008 Win2008..r2 Win2012 Win2012..r2
+            virtio_dev_iommu_platform = on
+            enable_guest_iommu = yes
+            virtio_dev_ats = on
+            machine_type_extra_params = "kernel-irqchip=split"
+            extra_params = "-device intel-iommu,intremap=on,eim=on,device-iotlb=on"
```

Comment:
in polarion test case, it's -device intel-iommu,intremap=on,device-iotlb=on

Reply:
in balloon case, it is "-device intel-iommu,intremap=on,eim=on,device-iotlb=on", pls have a check.

Reply:
update balloon test case.